app/src/main/java/com/xmartlabs/dd/ui/DashboardFragment.java

```diff
@@ -0,0 +1,111 @@
+package com.xmartlabs.dd.ui;
+
+import android.animation.ValueAnimator;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.github.florent37.diagonallayout.DiagonalLayout;
+import com.xmartlabs.dd.R;
+
+import butterknife.BindView;
+import butterknife.OnClick;
+
+/**
+ * Created by scasas on 3/31/17.
+ */
+@SuppressWarnings("deprecation")
+public class DashboardFragment extends BaseFragment {
+  private static final int ANIMATION_DURATION = 300;
+  private static final int QUANTITY_OF_OPTIONS = 3;
+
+  @BindView(R.id.bottom_black)
```

Comment:
This is kind of repetitive. Can we at least extract this to a different view?
(it would be even better to use a recyclerview)